packages/apps/marmalade-marketplace/src/components/CreateToken/index.tsx

```diff
@@ -59,40 +58,39 @@ function CreateTokenComponent() {
   });
 
   const [guardInput, setGuardInput] = useState({
-    uriGuard: walletKey,
-    burnGuard: walletKey,
-    mintGuard: walletKey,
-    saleGuard: walletKey,
-    transferGuard: walletKey,
+    uriGuard: account?.guard,
+    burnGuard: account?.guard,
+    mintGuard: account?.guard,
+    saleGuard: account?.guard,
+    transferGuard: account?.guard,
   });
 
   const [royaltyInput, setRoyaltyInput] = useState({
     royaltyFungible: 'coin',
-    royaltyCreator: webauthnAccount?.account ?? '',
-    royaltyGuard: walletKey,
+    royaltyCreator: account?.accountName ?? '',
+    royaltyGuard: account?.guard,
     royaltyRate: '0.05',
   });
 
   useEffect(() => {
-    if (webauthnAccount) {
-      const key = webauthnAccount.guard.keys[0];
-
+    if (account) {      
```

Comment:
I usually invert the if statements for early returns, so in this case,
```
if (!account) return

const.....
```

allows for less context that needs to be kept in mind while reading the code